src/app/storage/duck/sagas.ts

```diff
@@ -192,7 +233,7 @@ function* addStorageRequest(action) {
       // The objects that need to be rolled back are those that were fulfilled
       const rollbackObjs = storageAddResults.reduce((rollbackAccum, res) => {
         return res.state === 'fulfilled'
```

Comment:
Will this still work now that you're pushing actual results into the list instead of a promise?

Reply:
@eriknelson Good catch. totally forgot about this piece. will make changes